build-system/compile/bundles.config.extensions.json

```diff
@@ -711,6 +711,7 @@
     "options": {"hasCss": true}
   },
   {"name": "amp-video-iframe", "version": "0.1", "latestVersion": "0.1"},
+  {"name": "amp-video-iframe", "version": "1.0", "latestVersion": "0.1"},
```

Comment:
```suggestion
  {"name": "amp-video-iframe", "version": ["0.1", "1.0"], "latestVersion": "0.1"},
```

Reply:
I'll keep them separate since we actually need CSS.